sdk/inc/azure/core/az_mqtt5_rpc.h

```diff
@@ -26,14 +26,14 @@
  */
 #define AZ_MQTT5_RPC_DEFAULT_REQUEST_TOPIC_FORMAT                                     \
   "services/" _az_MQTT5_TOPIC_PARSER_SERVICE_ID_KEY "/" _az_MQTT5_RPC_EXECUTOR_ID_KEY \
```

Comment:
Nit: can we rename these to "_TOKEN" instead? (e.g. `_az_MQTT5_TOPIC_PARSER_SERVICE_ID_TOKEN`)?